### PR TITLE
Cron expression for delete job in aat

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-716

### Change description ###
Flux: https://github.com/hmcts/cnp-flux-config/pull/3105
Made it every 10 sec

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-processor/values.aat.template.yaml [hunk 1]
```diff
@@ -5,3 +5,4 @@ java:
   devApplicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
   environment:
     STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
+    DELETE_COMPLETE_FILES_CRON: "0/10 * * * * *"
```
Comment (nitinprabhu) [inline charts/bulk-scan-processor/values.aat.template.yaml:8]: this applies only to staging pod and not to all aat pods

Comment (apereverzin) [reply charts/bulk-scan-processor/values.aat.template.yaml:8]: @nitinprabhu should it be in values.yaml? in this case it will also be in prod - right?

Comment (nitinprabhu) [reply charts/bulk-scan-processor/values.aat.template.yaml:8]: for prod we can override in flux yaml if the value needs to be different. May be worth checking with team what we need.

I would day every 10 mins on prod and other environment 10 secs ?

Comment (lgonczar) [reply charts/bulk-scan-processor/values.aat.template.yaml]: I'd make sure flux for prod says every 10 minutes before this PR gets merged.

**Comment by lgonczar** [comment]
Please make sure flux change gets in first

**Pull request merged**